src/styles/variables.ts

```diff
@@ -217,4 +217,6 @@ export default {
     updateTextViewContainerWidth: 310,
     updateViewHeaderHeight: 70,
     workspaceProfileName: 20,
+    workspaceShareButtonHeight: 40,
```

Comment:
We should not be setting any kind of custom height or width on these buttons. Just use our standard button styles to achieve this.

Reply:
cc @Expensify/design

Reply:
@shawnborton removed custom height